specification/web/CertificateRegistration.Management/tspconfig.yaml

```diff
@@ -0,0 +1,51 @@
+parameters:
+  "service-dir":
+    default: "sdk/web"
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    omit-unreachable-types: true
+    emitter-output-dir: "{project-root}/.."
+    azure-resource-provider-folder: "resource-manager"
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/openapi.json"
+    examples-dir: "{project-root}/examples"
+    emit-lro-options: "all"
+  "@azure-tools/typespec-csharp":
+    flavor: azure
+    package-dir: "Azure.ResourceManager.Web"
```

Comment:
package-dir is deprecated for all the languages, use emitter-output-dir instead.